### DISCUSSION TITLE
[wayland] ONLYOFFICE desktop editors

### DISCUSSION THREAD

**Discussion by Sydiepus** [qtile/qtile#3597] [Q&A] [answered]
I have a problem with ONLYOFFICE desktop editors where when i open it it's hidden until i switch to floating layout.
I use columns as default.
If i switch back to columns the window disappear.
To make it use the columns layout :
- Switch to full screen.
- Switch to columns layout.
- Switch to normal... (unfullscreen).
Now it's working fine with columns layout and doesn't disappear.

It's working fine in sway and Qtile on X11 backend.

Should i open an issue about it ?

**Comment by m-col** [comment, marked as answer]
Interesting! Please do open an issue. It would be worth testing out on git master first, if you're running from the last release.

**Reply by Sydiepus** [reply]
Not much of an issue or better said not a Qtile issue but pywlroots in arch repos is out of date preventing Qtile from starting on the wayland backend. 

Error : 
```
ModuleNotFoundError: No module named 'wlroots.wlr_types.virtual_pointer_v1'
```

Sadly, i'm the one that requested this feature that is now preventing Qtile from starting.

**Reply by m-col** [reply]
If you're using the git version of Qtile, you might benefit from using the git version of pywlroots, or at least using pip so you can have more control over pywlroots version rather than depending on the arch devs. You shouldn't need the git version as nothing will be merged into qtile that isn't available in a pywlroots release.

**Reply by Sydiepus** [reply]
Thank you, I'll go with the pip version.  

I'll open an issue as soon as i test it on the git version.
